R/type-list-of.R

```diff
@@ -185,11 +185,7 @@ as.character.vctrs_list_of <- function(x, ...) {
 #' @method vec_ptype2 vctrs_list_of
 #' @export
 vec_ptype2.vctrs_list_of <- function(x, y, ..., x_arg = "x", y_arg = "y") {
-  if (inherits_only(x, c("vctrs_list_of", "vctrs_vctr"))) {
-    UseMethod("vec_ptype2.vctrs_list_of", y)
-  } else {
-    vec_default_ptype2(x, y, x_arg = x_arg, y_arg = y_arg)
-  }
+  UseMethod("vec_ptype2.vctrs_list_of", y)
```

Comment:
I guess you don't need `y` here now right?

Reply:
Right